API/Models/AccountModels/LoginModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace API.Models.AccountModels
+{
+    public class LoginModel
+    {
+        [Required]
+        [EmailAddress]
+        public string Email { get; set; }
+
+        [Required]
+        public string Password { get; set; }
+
+        public bool RememberMe { get; set; }
```

Comment:
Is `RememberMe` field really required in backend? Shouldn't this check be performed at frontend?
Removing this would allow us to reuse same DTO used for signup

Reply:
You're right, it's not really needed since we don't use cookies.